CalculateCurrentIssues.cls

```diff
@@ -1,71 +1,71 @@
- if (all_pub_new_issues.size()>2) {all_pub_new_issues[2].issue_status__c = 'Next+1';}
- if (all_pub_new_issues.size()>3) {all_pub_new_issues[3].issue_status__c = 'Next+2';}
- if (all_pub_new_issues.size()>4) {all_pub_new_issues[4].issue_status__c = 'Next+3';}
- if (all_pub_new_issues.size()>5) {all_pub_new_issues[5].issue_status__c = 'Next+4';}
- if (all_pub_new_issues.size()>7) {all_pub_new_issues[6].issue_status__c = 'Next+5';}
- if (all_pub_new_issues.size()>8) {all_pub_new_issues[7].issue_status__c = 'Next+6';}
- if (all_pub_new_issues.size()>9) {all_pub_new_issues[8].issue_status__c = 'Next+7';}
- if (all_pub_new_issues.size()>10) {all_pub_new_issues[9].issue_status__c = 'Next+8';}
- if (all_pub_new_issues.size()>11) {all_pub_new_issues[10].issue_status__c = 'Next+9';}
- if (all_pub_new_issues.size()>12) {all_pub_new_issues[11].issue_status__c = 'Next+10';}
- if (all_pub_new_issues.size()>13) {all_pub_new_issues[12].issue_status__c = 'Next+11';}
+ allPubNewIssues[0].issue_status__c = 'Current';
+ if (allPubNewIssues.size()>1) {allPubNewIssues[1].issue_status__c = 'Next';}
+ if (allPubNewIssues.size()>2) {allPubNewIssues[2].issue_status__c = 'Next+1';}
+ if (allPubNewIssues.size()>3) {allPubNewIssues[3].issue_status__c = 'Next+2';}
+ if (allPubNewIssues.size()>4) {allPubNewIssues[4].issue_status__c = 'Next+3';}
+ if (allPubNewIssues.size()>5) {allPubNewIssues[5].issue_status__c = 'Next+4';}
+ if (allPubNewIssues.size()>7) {allPubNewIssues[6].issue_status__c = 'Next+5';}
+ if (allPubNewIssues.size()>8) {allPubNewIssues[7].issue_status__c = 'Next+6';}
+ if (allPubNewIssues.size()>9) {allPubNewIssues[8].issue_status__c = 'Next+7';}
+ if (allPubNewIssues.size()>10) {allPubNewIssues[9].issue_status__c = 'Next+8';}
+ if (allPubNewIssues.size()>11) {allPubNewIssues[10].issue_status__c = 'Next+9';}
+ if (allPubNewIssues.size()>12) {allPubNewIssues[11].issue_status__c = 'Next+10';}
+ if (allPubNewIssues.size()>13) {allPubNewIssues[12].issue_status__c = 'Next+11';}
  
- update all_pub_new_issues;
+ update allPubNewIssues;
 
- list<issue__c> last2_pub_old_issues = [select id
+ list<issue__c> last2PubOldIssues = [select id
                                  from issue__c
-                                 where publication_name__c = :this_pub
+                                 where publication_name__c = :thisPub
                                  and deadline__c < :date.today()
                                  order by deadline__c desc
                                  limit 2];
 
-    if(last2_pub_old_issues.size()>=1) {last2_pub_old_issues[0].issue_status__c = 'Previous';}
-    if(last2_pub_old_issues.size()>=2) {last2_pub_old_issues[1].issue_status__c = 'Previous-1';}
-update last2_pub_old_issues;
+    if(last2PubOldIssues.size()>=1) {last2PubOldIssues[0].issue_status__c = 'Previous';}
+    if(last2PubOldIssues.size()>=2) {last2PubOldIssues[1].issue_status__c = 'Previous-1';}
+update last2PubOldIssues;
```

Comment:
* This DML statement is not surrounded by any try-catch block. Please add one to ensure exceptions are handled properly.
* This DML statement is inside a loop and is highly likely to hit governor limits. Please use collections to move the DML statement outside of the loop.